src/base_station/session.cpp

```diff
@@ -194,6 +196,11 @@ void Session::drive_sub_init(){
     motor_stat.first = "Motor Current (Amps)";
     motor_stat.second = 0.0;
     this->drive_sub_info.insert(motor_stat);
+
+    std::pair<std::string, double> throttle_stat;
+    throttle_stat.first = "Throttle Speed (km/s)";
```

Comment:
Do you know what the km/s unit means here? I'm uncertain on it means because the units used by the base station, [-255, 255], are mostly arbitrary and only mean ratio of the maximum speed. I know the other one uses km/s but I don't think it makes sense

Reply:
I dont know.. only put that there because of the speed and I figured it would be the same unit as the speed

Reply:
Would you like for me to edit this?

Reply:
> Would you like for me to edit this?

You could set the unit to something like (max 255) but not provide the unit. I don't think the km/s is relevant anymore

Reply:
did this